cmd/tracker-swebench/analyze.go

```
@@ -0,0 +1,1061 @@
+	}
+	// Order samples: setup, patch, harness.
+	for _, class := range []string{"setup", "patch", "harness"} {
+		dist.Samples = append(dist.Samples, samplesByClass[class]...)
+	}
+	return dist
+}
+
+// WriteAnalyzeText formats the report as the default human-readable output.
+// Uses plain ASCII tables — no external dependencies, safe for non-TTY pipes.
+func WriteAnalyzeText(w io.Writer, r *AnalyzeReport) error {
+	fmt.Fprintln(w, "tracker-swebench analyze")
```

golangci-lint (2.11.4)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)